### PR TITLE
Make the root view background color explicit

### PR DESCRIPTION
**Pull request by shaneosullivan**
The default white background for the root view causes an unattractive flash when loading any app with a non-white background.  Almost every developer has to search for how to fix it.  

This change makes the background color explicit in the file, so it can be easily change to match the color of the splash screen and the main screen without having to search for it.

**Test plan (required)**
Launched my app and it still has a white flash for the background before loading the app.  I changed the background colors to other values and the flash was that color.

### CODE REVIEW

**Comment by facebook-github-bot-4** [comment]
By analyzing the blame information on this pull request, we identified @geof90 and @tadeuzagallo to be potential reviewers.

**Comment by vjeux** [comment]
Sounds good to me, thanks!

**Comment by vjeux** [comment]
@facebook-github-bot shipit

**Comment by dhrrgn** [comment]
This will be very helpful to newer folks. However, it is still sort of hidden and people may not know to look there. It would be nice to have a "Customizing the Loading View" section in the docs.  This could include changing the background color, as well as the `setLoadingView` method (and maybe an example using an xib file for the loading view).

**Comment by vjeux** [comment]
@dhrrgn: this sounds like a great idea, would you be interested in spending an hour to write this and prevent so many future frustrations? :)

**Comment by shaneosullivan** [comment]
I figure everyone has to look into this file to change the js code location, so every dev will see it.  OK to land?

File: local-cli/generator-ios/templates/app/AppDelegate.m [hunk 1]
```diff
@@ -47,6 +47,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                                                       moduleName:@"<%= name %>"
                                                initialProperties:nil
                                                    launchOptions:launchOptions];
+  rootView.backgroundColor = [[UIColor alloc] initWithRed:1.0f green:1.0f blue:1.0f alpha:1];
```
Comment (modocache) [inline local-cli/generator-ios/templates/app/AppDelegate.m]: Drive-by comment: this is probably more tersely (and idiomatically) expressed as `[UIColor whiteColor]`.

Comment (shaneosullivan) [inline local-cli/generator-ios/templates/app/AppDelegate.m]: It is more terse, but that's not the point :-)  If you want to actually customize the colour, you need to see the Objective-C syntax, and hiding it all behind whiteColor was explicitly not done for that reason.

Comment (modocache) [inline local-cli/generator-ios/templates/app/AppDelegate.m]: I see your point. In that case, you could use `+[UIColor colorWithRed:green:blue:alpha:]`, which avoids the need to call `+[UIColor alloc]`.

Also, why use `1.0f` for every float value besides alpha, which is simply `1`?

All of these are nits of course, feel free to ignore. :)

**Comment by facebook-github-bot-7** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/1633702113622883/int_phab) to review.
